### PR TITLE
Fix green circle, create stories for icons, changed function to component.

### PR DESCRIPTION
**Pull request by besampson0**
### Description
Implements CASEFLOW-2462 - [https://vajira.max.gov/browse/CASEFLOW-2462](https://vajira.max.gov/browse/CASEFLOW-2462)

Fix the GreenCheckMark icon to be a true circle. Create two new stories to display all of the icons in RenderFunctions.jsx. Recreate loadingSymbolHtml (found in RenderFunctions.jsx) to be a component instead of a function and rename it LoadingSymbol. Replace it throughout the project with appropriate props being passed. Update snapshots that use LoadingSymbol.

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Circles in the Active Tasks section of the Case Details page are true circles
- [ ] Circles in the Case Timeline section of the Case Details page are true circles

### User Facing Changes

- Green circle.
Screen Shot 2021-10-22 at 10 17 55 AM

### CODE REVIEW

**Review by besampson0 — COMMENTED** [review]
Looks good.

File: client/app/components/RenderFunctions.jsx [hunk 1]
```diff
@@ -14,16 +14,16 @@ import { LOGO_COLORS } from '../constants/AppConstants';
  * @param {string} color The color of the non-gray part of the caseflow logo.
```
Comment (jcq) [inline client/app/components/RenderFunctions.jsx:14]: Now that these have moved to object props, the JSDoc param formats don't match the function signature anymore. Instead these would use the Parameters with Properties syntax. [See docs](https://jsdoc.app/tags-param.html).

The other (possibly better) option is to add comments to the PropTypes definitions, as that will then [display in the ArgsTable in Storybook](https://storybook.js.org/docs/react/writing-docs/doc-blocks#argstable).

**Review by jcq — APPROVED** [review]
On the whole, this looks great! I'm eager to see the rest of the icons documented in Storybook, though I understand why it was scoped out. 

I left one note about the documentation for the props; it's not a huge deal, but it'd be great to fully document them as long as we're bringing everything else up to code

File: client/app/components/RenderFunctions.jsx (hunk 1 above)
Comment (besampson0) [reply client/app/components/RenderFunctions.jsx:14]: Thanks JC, I am going to add PropTypes definitions for each of these in the next story that will add these to Storybook. Thanks!

**Pull request merged**